examples/guided_onboarding_example/public/components/step_two.tsx

```diff
@@ -53,17 +53,24 @@ export const StepTwo = (props: StepTwoProps) => {
         <EuiText>
           <p>
             <FormattedMessage
-              id="guidedOnboardingExample.guidesSelection.stepTwo.explanation"
+              id="guidedOnboardingExample.guidesSelection.stepTwo.explanation1"
               defaultMessage="The code on this page is listening to the guided setup state using an Observable subscription. If the state is set to
-              Search guide, step Browse documents, a EUI tour will be displayed, pointing to the button below."
+              Search guide, step Browse documents, a EUI tour will be displayed."
+            />
+          </p>
+          <p>
+            <FormattedMessage
+              id="guidedOnboardingExample.guidesSelection.stepTwo.explanation2"
+              defaultMessage="This page is used for the manual completion of Test guide, step 2. The manual completion popover
+              should appear on the header button 'Setup guide' to open the panel and mark the step done."
             />
           </p>
         </EuiText>
         <EuiSpacer />
         <EuiTourStep
           content={
             <EuiText>
-              <p>Click this button to complete step 2.</p>
+              <p>The tour step is displayed when Test guide, step 2 is active.</p>
```

Comment:
Can you delete the tour? I don't think it's necessary for step 2, but flashes briefly.